### PR TITLE
Env seed

### PR DESCRIPTION
**Pull request by jordan-schneider**
Adds the ability to set the gym environment random seeds, and makes SerialSampler do so on initialization. Other sampler should probably be given this functionality as well.

#136

### CODE REVIEW

**Comment by jordan-schneider** [comment]
I'm sorry, I don't see any suggestions at https://github.com/astooke/rlpyt/pull/137/files . Am I looking in the wrong place?

**Comment by astooke** [comment]
Hmm it looks like it started a "review", which should be visible in this conversation and in the changed file?

File: rlpyt/samplers/serial/sampler.py [hunk 1]
```diff
@@ -41,6 +42,12 @@ def initialize(
         """
         B = self.batch_spec.B
         envs = [self.EnvCls(**self.env_kwargs) for _ in range(B)]
+        if seed is None:
```
Comment (jordan-schneider) [inline rlpyt/samplers/serial/sampler.py:45]: asdf

**Comment by jordan-schneider** [comment]
Maybe I can't see the review because of permission issues? Maybe you forgot to hit the "submit review" button? Anyway, here's what my conversation looks like.

2020-04-06-171134_454x829_scrot

**Comment by astooke** [comment]
Indeed that was it, I had to submit it.  :)

**Comment by astooke** [comment]
OK great!  Looks like I didn't put the right indentation...oh you got that!

**Pull request merged**